client/js/render.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
+const $ = require("jquery");
+const templates = require("../views");
+const options = require("./options");
+const utils = require("./utils");
+const sorting = require("./sorting");
+
+const chat = $("#chat");
+const sidebar = $("#sidebar, #footer");
```

Comment:
Alrighty, so looking and this is for the buttons in the bottom of the sidebar (add server, help, sign out, etc) which are all still handled in lounge.js, so I have removed it from all the plugins (I checked that it didn't need them in each of them, they all seem to just be doing chan stuff).